net/keepalived/files/usr/bin/keepalived-rsync-inotify.sh

```diff
@@ -0,0 +1,14 @@
+#/bin/sh
+
+[ $# -lt 2 ] && exit 1
+
+peer=$1
+rsync_dir=$2
+INOTIFY_ACTIONS="create delete modify move moved_to moved_from"
+
+ARGS="${INOTIFY_ACTIONS:+ -e ${INOTIFY_ACTIONS// / -e }}"
+
+/usr/bin/inotifywait -q -r --exclude '/\..+' -m $rsync_dir $ARGS 2>/dev/null | while read -r dir action file; do
+	RSYNC_TARGET_DIR=$(echo "${dir}" | sed -e "s:${rsync_dir}::g")
+	ACTION=SYNC RSYNC_SOURCE="${dir}${file}" RSYNC_TARGET="${RSYNC_TARGET_DIR}${file}" /sbin/hotplug-call keepalived inotify
```

Comment:
Don't we have to check whether a sync has already been started?

Reply:
rsync is triggered from the peer(which is master), on the backup node, It only keeps watch on `sync_dir`.